phys2bids/io.py

```diff
@@ -69,18 +68,22 @@ def check_multifreq(timeseries, freq, start=0, leftout=0):
     return mfreq
 
 
-def process_labchart(channel_list, chtrig, header=[]):
+def generate_blueprint(channel_list, chtrig, interval, orig_units, orig_names):
     """
-    Process labchart header and channel_list and make a physio_obj.BlueprintInput.
+    Process items header items and channel_list and make a physio_obj.BlueprintInput.
 
     Parameters
     ----------
-    channel_list: list
+    channel_list : list of strings
         list with channels only
     chtrig : int
         index of trigger channel, starting in 1 for human readability
-    header: list
-        list with that contains file header
+    interval : list of strings
+        maximum sampling frequency or interval value and unit for the recording
```

Comment:
So this might be something like `["400Hz", "300Hz"]`?

Reply:
nope, more something like ``["400", "Hz"]`` or ``["1", "ms"]``  that's why i say for the recording, because in txt files the only information we have about frequency is the maximum. That's why we have the ``check_frequency`` function, to get the frequency for each channel

Reply:
Maybe an example helps - can you add it?